### DISCUSSION TITLE
Osu Lazer! + Wacom Driver between Area and clicks

### DISCUSSION THREAD

**Discussion by Yuzuctus** [ppy/osu#25404] [Q&A] [answered]
I wanted to play Osu!Lazer with my tablet but it didn't work with my Wacom settings.

I started to disable the drivers built into Lazer! 

It worked, I had the right Area but none of the buttons worked.

I tried to deactivate the high-precision mouse and my tablet no longer worked on Osu! 

So either I'm suffering from the Osu! drivers and I'm getting clicks (even when I press with the stylus, which is annoying for me) or I'm getting my area but not my clicks...

What can I do?

**Comment by bytenekos** [comment, marked as answer]
> suffering from the Osu! drivers and I'm getting clicks (even when I press with the stylus, which is annoying for me)

Have you disabled mouse using f10?
I'm using a wacom tablet just fine, both with lazers built in tablet driver, the regular tablet driver and opentabletdriver

**Reply by Yuzuctus** [reply]
I haven't disabled mouse clicks.

And I don't know if you can confirm, but it only happens when you're on a beatmap?

**Reply by bytenekos** [reply]
Try disabling mouse clicks, it only applies when in game (so if you disable them, they'll still work in song select and stuff, but will get turned off in game)

**Reply by Yuzuctus** [reply]
I've tested OpenTabletDriver, I'm having a bit of trouble configuring it as there are a lot more options, but in any case it works on Lazer.

**Reply by Yuzuctus** [reply]
Not very surprising given the amount of work you've done, but the last update solved all my problems.

big congratulations to you, you work really hard!
